docs/tileentities/tileentity.md

```diff
@@ -58,6 +58,18 @@ Achieving this is rather simple, by overriding following method
 !!! note
 	You should actually check the `BlockStates` and not just return `false` in order to prevent unwanted behavior and bugs.
     
+## Ticking `TileEntities`
+If you need a ticking `TileEntity` for example to calculate progress on a smelting process.
+You need to add the `ITickable` interface to your `TileEntity`.
+Now you can implement all your calculations within
+```JAVA
+	ITickable#update()
+```
+!!! note
+	This method is called 20 (twenty) times a second you should avoid having complicated calculations in here.
```

Comment:
More exactly, this method is called each tick. You should suggest mods cache the calculated results if something needs to be calculated only when there is a change, e.x. a recipe which changes only by input.

Reply:
Calculating the output of a recipe within the tick method is already bad practice, that shouldn't be done at all. But yeah, I changed a few things to be a little bit more clear.